main.go

```diff
@@ -87,6 +90,7 @@ func main() {
 		Log:              ctrl.Log.WithName("controllers").WithName("Rule"),
 		RuleConfigmap:    types.NamespacedName{Name: rulesConfigmapName, Namespace: rulesConfigmapNamespace},
 		ValidationConfig: validationConfig,
+		RulesFileName:    rulesFileName,
```

Comment:
I think there should be a basic validation here. I think K8s uses this regex for key names: `[-._a-zA-Z0-9]+` Controller should fail on startup (os.Exit(1)) if the parameter does not match the validation regexp